src/script/calling/enum/CallMessageType.ts

```diff
@@ -19,7 +19,8 @@
 
 export enum CALL_MESSAGE_TYPE {
   CANCEL = 'CANCEL',
-  CONF_START = 'CONFSTART',
+  CONF_START = 'CONFEND',
+  CONF_END = 'CONFSTART',
```

Comment:
Looks like a deliberate change, just want to make sure it's not something that got mixed up

Reply:
hm this is strange.

Reply:
yes, a typo, thanks for catching, fixed in https://github.com/wireapp/wire-webapp/pull/13449/commits/1d2e697c422bc6d209e1cc50d765821556eab373